joplin/static/css/preview.css

```diff
@@ -44,7 +43,7 @@
 .thumbnail,
 .thumbnail iframe {
   width: 100%;
-  height: 75vh;
+  height: 72vh;
```

Comment:
just highlighting this as a place where it would be nice to have Sass variables.